admin/README.md

```diff
@@ -2,3 +2,19 @@
 
 A Rails engine that provides an administrative interface to the Solidus ecommerce platform.
 
+## Development
+
+### Adding components to Solidus Admin
+
+When using the component generator from within the admin folder it will generate the component in the library
+instead of the sandbox application.
+
+```bash
+cd admin
+bin/rails g solidus_admin:component foo
```

Comment:
After reading https://github.com/solidusio/solidus/pull/5135/files#r1223834576, I'm not sure if this does need to have the namespace as well? Shouldn't it be

```
bin/rails g solidus_admin:component solidus_admin/foo
```

?

Reply:
Rails has a built-in mechanism (I think coming from the name-generator) that will add the namespace for engines automatically when called with an ENGINE_ROOT set. 

So this is correct, but adding it explicitly would work as well as there's also a check that won't add it if it's already there.